tasks/e2e/cypress/integration/01-tutorial/tutorial.spec.js

```diff
@@ -26,6 +26,14 @@ const BASE_DIR = Cypress.env('RW_PATH')
 describe('The Redwood Tutorial - Golden path edition', () => {
   // TODO: https://redwoodjs.com/tutorial/saving-data
   // TODO: https://redwoodjs.com/tutorial/administration
+  after(() => {
+    cy.exec(
+      `cd ${BASE_DIR}; git add . && git commit -a --message=01-tutorial`,
+      {
+        failOnNonZeroExit: false,
```

Comment:
We should probably make it fail if this doesn't work.

Reply:
Good idea. I don't believe it's critical, but the logic makes more sense! Doing now.